src/test/util/setup_common.cpp

```diff
@@ -276,15 +276,15 @@ TestChainSetup::TestChainSetup(int num_blocks, const std::vector<const char*>& e
             /* TestChain100Setup */
             {  100, uint256S("0x6ffb83129c19ebdf1ae3771be6a67fe34b35f4c956326b9ba152fac1649f65ae") },
             /* TestChainDIP3BeforeActivationSetup */
-            {  430, uint256S("0x20c025ce708233d05280099f82f09def00ed5aaf5430316070e0dd8807897c41") },
+            {  430, uint256S("0x0bcefaa33fec56cd84d05d0e76cd6a78badcc20f627d91903646de6a07930a14") },
```

Comment:
these hashes are changed because v19 is activated earlier (under block 1000), right?

if so, can we make v20 on RegChain to activate earlier to speed-up functional tests?

Reply:
they changed because there is no signalling now, v19 activation height is the same